aws/cloudformation-templates/template.yaml

```diff
@@ -466,6 +466,10 @@ Parameters:
     Default: 'nr50Qdu7FM94n2X1GYuhA8cFzyvdYTJ5Qka4XMOd'
     NoEcho: true
 
+  ACMCertificateArn:
```

Comment:
this parameter should be added in the "Optional Integrations" group to avoid having it floating at the end (it looks like it's part of the Fenix integration)

And add a comment that is optional only required for deployment with a custom CNAME domain